paddle/fluid/operators/mkldnn/matmul_mkldnn_op.cc

```diff
@@ -158,7 +158,8 @@ class MatMulV2MKLDNNHandler
 
     // TODO(jczaja): Why not for int8??
```

Comment:
It would be great if @jczaja could answer this question :)

Reply:
it would be great:)

Reply:
It seems that there is no reason why it can't be done for int8, please remove this comment and create a ticket for it.